### ISSUE TITLE
Error when trying to merge a jpg and a mp4 file.

### ISSUE THREAD

**Issue by risako11** [PetrVys/MotionPhoto2#9]
Hello I am trying to get this to work, but for some reason I am getting an error. Am I doing something wrong? 
I installed 3.13.0 python using this file "python-3.13.0-amd64.exe" and I installed ExifTool using this file "ExifTool_install_12.99_64.exe". 
Then I opened terminal in the MotionPhoto2 folder and entered this command: `pip install -r requirements.txt`

After installing requirements I typed this: `python motionphoto2.py --input-directory c:\Users\rakak\Downloads\MotionPhoto2\test\`

These are the files I am trying to merge:
![image](https://github.com/user-attachments/assets/4c53da12-ae18-4871-b678-e3abb004a9ae)

And this is the result:
![image](https://github.com/user-attachments/assets/603b8b52-31cb-4bba-9932-50e6810e1a15)

Am I missing something?

**Comment by PetrVys** [comment]
Please try again with the latest fix. Your MP4 file most likely does not come from iphone (duh, only mov files do) and thus it was not possible to get the iphone keyframe of the start of the motion photo. Default is now used instead of crash.

**Comment by risako11** [comment]
Thank you it works now :) The photo was from Samsung Galaxy S22 Ultra.

Here is my log just for information:
![image](https://github.com/user-attachments/assets/0504c78d-7bfb-498b-a563-1fe4b66465aa)

(Prekvapilo ma, že práve český brat má tento projekt na svedomý👍)

**Comment by PetrVys** [comment]
Just out of curiosity - what's the genesis of the input file you tested this with (test.jpg)? Given the XMP tags copied, it seems it was already a motion photo?

**Comment by risako11** [comment]
I used Google Takeout function to download all my photos from Google Photos. For some reason Google Takeout disconnects the video from live photo and puts it as separate mp4 file. So I have a lot of jpg+mp4 files that need to be joined together...However all HEIF files I have saved in Google Photos were downloaded correctly. Those files don't need to be joined together.
I also realized that if I download my photos from Google Photos manually, without using Google Takeout, then the jpg downloads correctly and I don't have to join it together with a separate mp4 file...Its just the Google Takeout that creates this problem with live jpg files.

**Comment by PetrVys** [comment]
Ah, makes sense. I'll probably write about it in the readme, that may be useful to others...

Diky a at to slouzi 👍

**Comment by risako11** [comment]
Ok, so just now, I realized that the jpg files from Google Takeout were actually animated all along...

I thought they were static, because when I uploaded them directly into an album on Google Photos, they became static. 
But for some reason, when you upload dynamic photos just generally into Google Photos, not into a specified album, they will upload correctly, Thank you Google for confusing me and wasting my time...

When I was opening those files in windows photo viewer I swear the button to enable motion wasn't showing also, so it confirmed my assumptions, that the images are static, but maybe the button just didn't load quickly enough and then I didn't notice it? I am utterly confused. 

At least this script now works with jpgs, so it wasn't for nothing lol.
